index.js

```diff
@@ -228,6 +287,10 @@ function printComplexValue(val, indent, prevIndent, spacing, edgeSpacing, refs,
     return hitMaxDepth ? '[Map]' : printMap(val, indent, prevIndent, spacing, edgeSpacing, refs, maxDepth, currentDepth, plugins, min, callToJSON, printFunctionName);
   } else if (toStringed === '[object Set]') {
     return hitMaxDepth ? '[Set]' : printSet(val, indent, prevIndent, spacing, edgeSpacing, refs, maxDepth, currentDepth, plugins, min, callToJSON, printFunctionName);
+  } else if (isHTMLElement(val, toStringed)) {
```

Comment:
As mentioned in my other comment, would you mind moving this into a dom plugin that we could add to later on?

You can see some examples here: https://github.com/thejameskyle/pretty-format/tree/master/plugins